component/src/main/java/io/siddhi/extension/store/mongodb/util/MongoTableUtils.java

```diff
@@ -646,7 +647,8 @@ private static SocketFactory extractSocketFactory(
         }
 
         try {
-            SSLContext sslContext = SSLContext.getInstance("SSL");
+            SSLContext sslContext =
+                    SSLContext.getInstance(configReader.readConfig(MongoTableConstants.ENCRYPTION_PROTOCOL, "TLSv1.2"));
```

Comment:
Isn't the default value SSL according to doc annotation

Reply:
https://github.com/siddhi-io/siddhi-store-mongodb/pull/61/commits/be04a98ace2f11b6ba59ad1333899db3d6025ac9